yarn-project/aztec/src/cli/cmds/start_archiver.ts

```diff
@@ -24,6 +31,8 @@ export async function startArchiver(
     'archiver',
   );
 
+  await validateL1Config({ ...getArchiverConfigFromEnv(), ...archiverConfig });
```

Comment:
since this is a CLI call, we shouldn't need `getArchiverConfigFromEnv` since all these env vars are parsed by commanderjs if a CLI option hasn't been passed